client/components/domains/domain-search-results/style.scss

```diff
@@ -23,3 +23,22 @@
 		}
 	}
 }
+
+.domain-search-results__transfer-card {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	font-size: 17px;
+}
+
+.domain-search-results__transfer-card-copy {
+	margin-left: 10px;
+	p {
+		padding-top: 10px;
+		margin-bottom: 0;
+	}
+}
+
+.domain-search-results__transfer-card-link {
+	min-width: 190px;
```

Comment:
This should probably be `100px` (to match the size of the "Updgrade >" links in the sections below) on screens below ~375px wide otherwise the link text will be cut off on screens like the iPhone SE/5/5s (https://cloudup.com/c0Ci_yMToIB).